src/main/scala/com/mwz/sonar/scala/sensor/Measures.scala

```diff
@@ -21,10 +21,15 @@ package com.mwz.sonar.scala.sensor
 import scala.annotation.tailrec
 import scalariform.lexer.{Token, Tokens}
 
+import scala.util.matching.Regex
```

Comment:
Maybe I'm being too fussy, but the extra space before this import is unnecessary.
Also we should had the imports sorted alphabetically, so this should be before the scaliform one?, what do you think ?

Reply:
I forgot to reorganise those imports, fixed in a5f7bb8.